vault/changelog.early-seed.md

```diff
@@ -10,27 +10,32 @@ nav_exclude: true
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.99.0
+## 1.00.0
 
-Dendron 0.99 has sprouted 🌱
+Dendron 1.00 has sprouted 🌱
 
-**Better Task Notes**: Get things done - task notes now have custom commands to set their status and mark them as complete.
-More information: [[Set Task Status|dendron://dendron.dendron-site/dendron.ref.commands#set-task-status]]
+**v100:** We will be releasing version 100 of Dendron next week! This will mark 100 releases and over 600 features shipped! We'll be launching on product hunt on Tuesday June 21st, 12:01AM PST. If Dendron has been of use to you, please leave feedback on our [product hunt page](https://www.producthunt.com/posts/dendron-v100) at that time - this would help us out a lot! 🙏
 
-**Obsidian Import**: Vault to vault - we now have a dedicated command to import your notes from obsidian into Dendron!
-More information: [[Import Obsidian Vault|dendron://dendron.dendron-site/dendron.ref.commands#import-obsidian-vault]]
+**Recent Workspaces Panel:** Navigating through multiple workspaces has never been easier! This new panel will show you your last 5 opened Dendron workspaces and you can access them with just a click.
+- More information: [[Recent Workspaces|dendron://dendron.dendron-site/dendron.topic.sidebar.recent-workspaces]]
+
+**BREAKING:** Handlebars are now set as the default template format. Users that were using <% %> syntax for template variables will no longer work. Please see the [[Migration|dendron://dendron.dendron-site/dendron.topic.templates.handlebars.migration]] guide to switch to handlebar based templates. 
+
+**BREAKING:** `Insert Note` command no longer exists. It is replaced by the `Apply Template` command to [[Create a template|dendron://dendron.dendron-site/dendron.topic.templates.handlebars#create-a-template]]. 
```

Comment:
The `Insert Note` command has now been removed. Use [[Apply Template|dendron://dendron.dendron-site/dendron.topic.templates.commands.apply-template]] command instead